src/app_model/backends/qt/_qkeymap.py

```diff
@@ -76,109 +89,111 @@ def __init__(self, kb: KeyBinding) -> None:
         super().__init__(*ints)
 
 
+qt_keys = Qt.Key if PYQT6 else Qt
```

Comment:
can you comment on the need for this?  tests do cover this file, and run on PySide 2/6 PyQt5/6 from 5.12 on (which is when the scoped Qt.Key was introduced) ... where were you running into issues

Reply:
ah, i was copying this [from vispy](https://github.com/vispy/vispy/blob/6b9d39c8c7ab75ceb7f15ccccef5f0d27eaeeec1/vispy/app/backends/_qt.py#L153), i thought that they were right since the keys didn't appear in `dir(Qt.Key)` but upon further inspection they are implemented in the `__getattr__`. will revert